### ISSUE TITLE
On whihc chain is this web app running ??? (https://app.lokian.monster/)

### ISSUE THREAD

**Issue by Lord-Ravan** [tytzM17/Lokian.eth#7]
Kindly assist me how can I perform transaction on the webapp that has been made by you guys.

What network do I need to use while performing transaction of your web app (https://app.lokian.monster/) 
?

**Comment by tytzM17** [comment]
hello, we are on polygon, avalanche, moonbeam, astar {maybe buggy} and fantom here is our website for further info www.lokian.monster

**Comment by Lord-Ravan** [comment]
Am I able to use the app on any test net?

**Comment by tytzM17** [comment]
yes, we have it on polygon mumbai

**Comment by Lord-Ravan** [comment]
From where can I get LOK token ?

When ever I am trying to buy something like sword, shields, etc. I am facing this issue.
![issue](https://user-images.githubusercontent.com/89789689/222900280-efe82ac6-dae8-4cdc-b333-164e38eb7754.JPG)

**Comment by tytzM17** [comment]
yeah, coz that can only be bought by LOK erc20 token, what is your pubkey address and ill send you some LOk tokens

**Comment by Lord-Ravan** [comment]
well sir I am unaware about pubkey address, from where can I get it and how much do I need to pay for it ?

**Comment by tytzM17** [comment]
no need to pay, we have some liquidity on sushiswap on polygon but ill just give you some

**Comment by Lord-Ravan** [comment]
0x34764E7Cbe5Aed0224615ee43251d41215a5069E will this one work ?

**Comment by tytzM17** [comment]
'one eternity later' // i have now sent u some Loki coins (erc20) on both polygon mainnet and mumbai. thank u
